src/m3/serializer.ts

```diff
@@ -39,16 +47,13 @@ export const serializeMetamodel = (metamodel: Metamodel): SerializedNode[] => {
                 references: {
                     [metaFeatures.concept_extends.id]: asRefIds(thing.extends),
                     [metaFeatures.concept_implements.id]: asIds(thing.implements)
-                },
-                parent: thing.parent?.id
+                }
             })
             thing.features.forEach(visit)
             return
         }
         if (thing instanceof ConceptInterface) {
-            json.push({
-                type: metaConcepts.conceptInterface.id,
-                id: thing.id,
+            serializeNode(metaConcepts.conceptInterface, {
```

Comment:
I thought we removed the custom serializer. Is that part of a separate PR?

Reply:
The generic serializer is still in the works, mostly because of where the DIM2 interacts with PM2: you need to parametrize the generic serializer with a mapping JS object -> M2 element.